### ISSUE TITLE
One hub with two Tempest devices

### ISSUE THREAD

**Issue by PianSom** [briis/weatherflow_forecast#192]
I have a single Weatherflow Tempest hub (Station) which has two Tempest Devices using it. Only one is found by the integration. Is that expected? If not, is it possible to add support for multiple Devices?

**Comment by briis** [comment]
Yes that is expected. I have only ever had 1 Tempest to 1 Hub, so I did not have a chance to test out that scenario.
I am pretty sure it could be supported, but for me to work on it, I would need access to your installation and your data. Meaning I would need your Station ID and API Key.

If you are willing to give me that, I will give it a try. Please note, I am on vacation all next week, so I would not be able to start any work on it before after that. And even then, it might take a little while, as I am afraid I not to rewrite some base logic.

If you want to continue, you can email the desired info to me directly, so it is not posted here: bjarne@briis.com

**Comment by PianSom** [comment]
Thanks. I'll happily share access. I will contact you on your return. 

Before that I will endeavour to stabilise the unit that is not currently detected by Weatherflow Forecast - it's been having some power issues during the recent period when there has been minimal sun.

**Comment by chheiss** [comment]
Any news here? I have two stations, but can not use the second anymore, as the cloud integration is throwing this error: 

Failed setup, will retry: 'pressure_trend'

**Comment by briis** [comment]
Nothing yet. I have several Integrations I maintain, and right now I am playing catch-up with the changes the HA team is doing to the underlying structure of Home Assistant. But you have not been forgotten and I will get to it as soon as I can.

**Comment by PianSom** [comment]
Hi @briis - the worst of winter is hopefully behind us and my weaker Tempest is now showing signs of life most of the time. Are you still able to have a look at the dual Tempest issue? If this is likely to reach the top of your list any time soon I will email over the sets you asked for.

**Comment by PianSom** [comment]
Bumping @briis - still willing and able when you are ready. Let me know.

**Comment by briis** [comment]
This is an old issue, so closing it now. Please re-open if it is still valid.
